components/SortControl.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from '../styles/SortControl.module.css';
+
+export interface SortControlProps {
+    onSelectedSortChange: (newValue: number) => void;
+}
+
+export default function SortControl(props: SortControlProps) {
+    const onSelectedSortChange = React.useCallback(
+        (cb: React.ChangeEvent<HTMLSelectElement>) => {
+            props.onSelectedSortChange(parseInt(cb.currentTarget.value));
+        },
+        [props.onSelectedSortChange]
```

Comment:
Technically this is wrong, and I need to change FilterControl to do this too. Destructure the props object first, then add it as a dependency.

Reply:
ill probably bug you to figure out what you mean for this one soon